media-video/shotcut/shotcut-20.10.31.ebuild

```diff
@@ -3,59 +3,47 @@
 
 EAPI=7
 
-inherit flag-o-matic qmake-utils xdg
+inherit qmake-utils xdg
 
 DESCRIPTION="A free, open source, cross-platform video editor"
-HOMEPAGE="https://www.shotcut.org/ https://github.com/mltframework/shotcut/"
+HOMEPAGE="https://www.shotcut.org https://github.com/mltframework/shotcut"
 SRC_URI="https://github.com/mltframework/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="GPL-3+"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
-BDEPEND="
-	dev-qt/linguist-tools:5
-"
-COMMON_DEPEND="
-	dev-qt/qtcore:5
+BDEPEND="dev-qt/linguist-tools:5"
+
+COMMON_DEPEND="dev-qt/qtcore:5
 	dev-qt/qtdeclarative:5[widgets]
 	dev-qt/qtgui:5
 	dev-qt/qtmultimedia:5
 	dev-qt/qtnetwork:5
 	dev-qt/qtopengl:5
 	dev-qt/qtprintsupport:5
 	dev-qt/qtsql:5
-	dev-qt/qtwebkit:5
+	dev-qt/qtwebengine:5
 	dev-qt/qtwebsockets:5
 	dev-qt/qtwidgets:5
 	dev-qt/qtxml:5
-	>=media-libs/mlt-6.18.0[ffmpeg,frei0r,jack,qt5,sdl,xml]
-	media-libs/webvfx
-	media-video/ffmpeg
-"
+	>=media-libs/mlt-6.22.1[ffmpeg,frei0r,jack,melt,qt5,sdl,xml]
+	media-video/ffmpeg"
+
 DEPEND="${COMMON_DEPEND}
 	dev-qt/qtconcurrent:5
-	dev-qt/qtx11extras:5
-"
+	dev-qt/qtx11extras:5"
+
 RDEPEND="${COMMON_DEPEND}
 	dev-qt/qtgraphicaleffects:5
 	dev-qt/qtquickcontrols:5
-	virtual/jack
-"
-
-src_prepare() {
-	default
-
-	sed -i -e '/QT.*private/d' \
-		src/src.pro || die
```

Comment:
This is no longer necessary?

Reply:
It builds fine without in my case - The line which got removed is:
```QT       += qml-private core-private quick-private gui-private``` (Using ```/usr/lib64/qt5/mkspecs/modules/qt_lib_{qml,core,quick,gui}_private.pri```)
Don't know what has been the problem - maybe thoses mkspecs or headers were missing in earlier QT version?

Reply:
Traced it back to this PR comment: https://github.com/gentoo/gentoo/pull/14393#issuecomment-578481918

Basically silencing a bogus warning.